qds_sdk/connection.py

```diff
@@ -50,11 +50,11 @@ def __init__(self, auth, rest_url, skip_ssl_cert_check, reuse=True):
             self.session_with_retries = requests.Session()
             self.session_with_retries.mount('https://', MyAdapter(max_retries=3))
 
-    @retry((RetryWithDelay, requests.Timeout), tries=6, delay=30, backoff=2)
+    @retry((RetryWithDelay, requests.Timeout, ServerError), tries=6, delay=30, backoff=2)
     def get_raw(self, path, params=None):
         return self._api_call_raw("GET", path, params=params)
 
-    @retry((RetryWithDelay, requests.Timeout), tries=6, delay=30, backoff=2)
+    @retry((RetryWithDelay, requests.Timeout, ServerError), tries=6, delay=30, backoff=2)
```

Comment:
This will apply to all network call? @chattarajoy

Reply:
only get requests. Retrying put/post/delete requests might case issues, hence added these to get requests only.